builtins/amp-ad.md

```diff
@@ -129,6 +129,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [Yieldmo](../ads/yieldmo.md)
 - [YahooJP](../ads/yahoojp.md)
 - [yieldone](../ads/yieldone.md)
+- [Zucks](../ads/zucks.md)
```

Comment:
This file is going to to be deleted. No change needed. It's moved to `/extensions/amp-ad/amp-ad.md`